remind_mfa/cement/cement_mfa_system_future.py

```diff
@@ -1,48 +1,70 @@
 import numpy as np
 import flodym as fd
+from enum import Enum
 
 from remind_mfa.common.assumptions_doc import add_assumption_doc
+from remind_mfa.cement.cement_carbon_uptake_model import CementCarbonUptakeModel
+
+
+class CementMode(str, Enum):
+    base = "base"
+    carbon_flow = "carbon_flow"
```

Comment:
variable names of constants could be in all caps: https://peps.python.org/pep-0008/#constants (not the strings though)